TombEngine/Scripting/Internal/ReservedScriptNames.h

```diff
@@ -339,6 +339,8 @@ static constexpr char ScriptReserved_FadeOutComplete[]				= "FadeOutComplete";
 static constexpr char ScriptReserved_SetCineBars[]					= "SetCineBars";
 static constexpr char ScriptReserved_SetFOV[]						= "SetFOV";
 static constexpr char ScriptReserved_GetFOV[]						= "GetFOV";
+static constexpr char ScriptReserved_SetCameraRoll[]				= "SetObjectCameraRoll";
```

Comment:
Does roll setting/getting only works in object camera mode? Can we make it work in other modes as well? Otherwise this API looks a little inconsistent (which underlines general problem with lack of game camera API though).